### ISSUE TITLE
"Debug Console" missing in Quick Open View list after dragged from Panel to Side Panel

### ISSUE THREAD

**Issue by zoo7314** [microsoft/vscode#142464]
Issue Type: <b>Bug</b>

1. command: Quick Open View,  "Debug Console" present
2. drag "Debug Console" to Side Panel
3. command: Quick Open View, "Debug Console" missing

VS Code version: Code 1.64.0 (5554b12acf27056905806867f251c859323ff7e9, 2022-02-03T04:20:17.224Z)
OS version: Darwin x64 19.6.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 1|
|Memory (System)|16.00GB (4.40GB free)|
|Process Argv|-psn_0_21279818 --crash-reporter-id cc136930-0fe0-43e4-ac43-316a8ca1a6c9|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|11.7.0
vetur|oct|0.35.0
LiveServer|rit|5.7.4
partial-diff|ryu|1.4.3
open-in-browser|tec|2.0.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392cf:30425750
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30433059
pythonvs932:30410667
wslgetstartedc:30433508
vs360:30404995
vsrem710cf:30416617

```

</details>

**Comment by roblourens** [comment]
There are two entries for it now

<img width="637" alt="image" src="https://user-images.githubusercontent.com/323878/155798165-2cc27b2b-7bf1-41d5-a376-5c425640f8e4.png">
